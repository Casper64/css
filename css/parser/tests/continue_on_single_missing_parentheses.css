.test {
    width: calc(100vw - (20px * 30px);
    color: rgba( 0 0 0;
    width: calc(100vw - (20px * 30px));
}