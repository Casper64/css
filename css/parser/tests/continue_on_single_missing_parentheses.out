css/parser/tests/continue_on_single_missing_parentheses.css:2:38: error: unexpected ";" inside a function, expecting ")". Did you forget to close the parentheses?
    1 | .test {
    2 |     width: calc(100vw - (20px * 30px);
      |                                      ^
    3 |     color: rgba( 0 0 0;
    4 |     width: calc(100vw - (20px * 30px));
css/parser/tests/continue_on_single_missing_parentheses.css:3:23: error: unexpected ";" inside a function, expecting ")". Did you forget to close the parentheses?
    1 | .test {
    2 |     width: calc(100vw - (20px * 30px);
    3 |     color: rgba( 0 0 0;
      |                       ^
    4 |     width: calc(100vw - (20px * 30px));
    5 | }